earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -30,6 +30,9 @@
 #include "common/SharedString.h"
 #include "StorageManagerAssetHandle.h"
 #include "AssetSerializer.h"
+#include "AssetFactory.h"
+
+using namespace AssetFactory;
```

Comment:
Generally speaking it's not a good idea to have a `using namespace` statement in a header file. I can understand why it makes life easier here, though.